_layouts/base.html

```diff
@@ -159,11 +156,14 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
         }
       }
     </script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.2.0/anchor.min.js"
-            integrity="sha256-m1eTvwEHwmsw4+XKF7BshClVJEPwTVycveNl0CS0Mkk="
+    <script src="https://cdn.jsdelivr.net/npm/anchor-js@4.3.1/anchor.min.js"
+            integrity="sha256-0WMZ9PF4b2hTF66Eglv/9H5Vwk6lnOG4AbmCJxo96WQ="
             crossorigin="anonymous"
             onload="loadAnchors()" async></script>
     <script src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
```

Comment:
Should there be an `integrity` attribute here, too?

Reply:
We can only use an `integrity` attribute with files that won't/shouldn't change. In this case, we're simply loading the latest 2.x version of the DocSearch library, so the checksum would change when a new version is released. To use a brew analogy, this is like how we can't use `sha256` with unversioned Casks.